Makefile

```diff
@@ -286,13 +286,14 @@ build-rpms-redistributable:
 	hack/build-rpm-release.sh
 .PHONY: build-rpms-redistributable
 
-# Build a release of OpenShift using tito for linux/amd64 and the images that depend on it.
+# Build a release of OpenShift (all the RPMs and images for all platforms)
 #
 # Args:
 #
 # Example:
 #   make release-rpms
-release-rpms: clean build-rpms
+release-rpms: clean build-rpms-redistributable
+	hack/build-base-images.sh
```

Comment:
> Oh -- the flow now is build RPMs, build base images with mounted RPMs, etc -- we do need RPMs to be present before the image builds

`make release` is also building images and now we'll require the RPMs to be present. Should we also change that?